tasks/debian10.yml

```diff
@@ -0,0 +1,13 @@
+---
+- name: Install Pacemaker cluster packages to all nodes
+  apt:
+    name: "{{ cluster_node_is_remote | bool | ternary(pacemaker_remote_packages, pacemaker_packages) }}"
+    state: 'present'
+
+- name: Check if Corosync configuration is default configaration
+  shell: '/usr/bin/dpkg --verify corosync'
```

Comment:
(medium) - could you please use `command` module instead of `shell` as there is no need for shell expansion?

Reply:
Yes sure I should be done now

Reply:
looks good, this is resolved.